### ISSUE TITLE
Does CephFS provisioner support resizing?

### ISSUE THREAD

**Issue by NahaliY** [kubernetes-incubator/external-storage#1154]
I use CephFS provisioner to provision PersistentVolume from PVC. 
Is there any possible way to adjust the size of the volume,  or using quotas ?

**Comment by wongma7** [comment]
No, https://github.com/ceph/ceph-csi might, please ask there as well
